scripts/getsamplesummary.py

```diff
@@ -7,10 +9,11 @@
 import re
 import socket
 import os
+import os.path
```

Comment:
I've always wondered why you would need to do this actually? Isn't is equivalent to:

```python
import os

# ...

os.path.join(some_path, some_filename)
```